src/backends/github.rs

```diff
@@ -13,6 +13,12 @@ use tempdir;
 
 use crate::{confirm, errors::*, get_target, version, Download, Extract, Move, Status};
 
+// Used to construct the real binary name on Windows.
+#[cfg(target_os = "windows")]
+const EXECUTABLE_SUFFIX: &str = ".exe";
+#[cfg(not(target_os = "windows"))]
+const EXECUTABLE_SUFFIX: &str = "";
```

Comment:
If you're still pursuing this PR, you can use [`std::env::consts::EXE_SUFFIX`](https://doc.rust-lang.org/std/env/consts/constant.EXE_SUFFIX.html) instead of rolling your own constant for this.